src/components/Table.js

```diff
@@ -138,7 +136,7 @@ export function Table({
     const headingEl = headerRef.current;
     const scrollboxEl = scrollboxRef.current;
 
-    if (rowEl && headingEl && scrollboxEl) {
+    if (rowEl) {
```

Comment:
Because `scrollboxRef` and `headerRef` are not longer as a mutable ref, whose `current` value can change between `null` to another type, we don't have to test if they are null.